### ISSUE TITLE
Resolving cookbook dependencies error

### ISSUE THREAD

**Issue by iamajvillalobos** [TalkingQuickly/rails-server-template#52]
Running on Mac OS Sierra I encountered this problem, installed Chef and Knife, went straight to Chapter 4, clone your repo and run `berks vendor`. I got this error instead:

```
rails-server-template master % berks vendor
Resolving cookbook dependencies...
Fetching 'nginx' from git://github.com/miketheman/nginx.git (at master)
Git error: command `git reset --hard e36944b0dac1f21044a4f0d01fc566023325cfde` failed. If this error persists, try removing the cache directory at '/Users/ajvillalobos/.berkshelf/.cache/git/a7e1b73c3018df83378b1e5e78affe3e5399e301'.Output from the command:

fatal: Could not parse object 'e36944b0dac1f21044a4f0d01fc566023325cfde'.

rails-server-template master %
```

I'm not sure what's wrong, any thoughts?

EDIT: 

Saw this [ticket](https://github.com/berkshelf/berkshelf/issues/1634) in [Berkshire Repo](https://github.com/berkshelf/berkshelf), I'm not sure if there's a fix?

**Comment by jr180180** [comment]
Hey @iamarmanjon , I received the same thing. I was able to find the cookbook in the Chef Supermarket at https://supermarket.chef.io/cookbooks/nginx

I commented out the:

`cookbook 'nginx', github: 'miketheman/nginx'`

and replaced it with:

`cookbook 'nginx', '~> 2.7.6'`

When I ran "berks vendor," it pulled through.

We should note that the repo may have been abandoned: https://github.com/miketheman/nginx/issues/432

I was also getting the error at https://github.com/berkshelf/berkshelf/issues/1669. According to https://github.com/berkshelf/berkshelf/pull/1668, it looks like the next shipment of ChefDK will fix that. 

Hope that helps you out, bud!

**Comment by git-toni** [comment]
Additionally don't forget to `rm Berksfile.lock` and `rm -r ~/.bershelf/.cache/git*`

**Comment by rubygeek** [comment]
thanks, this fixed my problem. This should be accepted and merged https://github.com/TalkingQuickly/rails-server-template/pull/54

**Comment by TalkingQuickly** [comment]
Thanks, this is now resolved as part of the 5.0 upgrade which supports Ubuntu 20.04
